lib/server.js

```diff
@@ -1139,11 +1139,49 @@ WalletService.prototype._getBalanceFromAddresses = function(addresses, cb) {
 WalletService.prototype.getBalance = function(opts, cb) {
   var self = this;
 
-  self.storage.fetchAddresses(self.walletId, function(err, addresses) {
+  function getActiveAddresses(cb) {
+    async.parallel([
+
+      function(done) {
+        self.storage.fetchRecentAddresses(self.walletId, (Date.now() / 1000) - 100, done);
```

Comment:
Since helpers are just for testing, I think we are going to need a default value in `defaults.js`. Any ideas on what would make a good default value?